bears/python/VultureBear.py

```diff
@@ -32,14 +63,10 @@ def run(self):
 
         See <https://bitbucket.org/jendrikseipp/vulture> for more information.
         """
-        stdout_output, _ = run_shell_command(
-            (self.EXECUTABLE,) +
-            tuple(filename for filename in self.file_dict.keys()),
-            cwd=self.get_config_dir())
-
-        for match in re.finditer(self.OUTPUT_REGEX, stdout_output):
-            groups = match.groupdict()
+        filenames = list(self.file_dict.keys())
+        unused = retrieve_unused(filenames)
+        for unused_entity in unused:
```

Comment:
Do you mean add code for removing unused imports into vulture?

If so, we have [autoflake](https://github.com/myint/autoflake/) for that, and a coala GSOC project will be doing crazy enhancements like https://gitlab.com/coala/GSoC-2017/issues/216